src/app/booking/ActivitiesCalendar.js

```diff
@@ -230,6 +233,78 @@ export function ActivityCard({ activity, targetDate }) {
   );
 }
 
+export function EmblaContainer({ emblaSx: emblaSxOuter, children }) {
+  const [viewportRef, embla] = useEmblaCarousel({ align: "start", loop: true });
+  const { selectedIndex, scrollSnaps, onDotButtonClick } = useDotButton(embla);
+
+  const scrollPrev = useCallback(() => {
+    if (embla) embla.scrollPrev();
+  }, [embla]);
+
+  const scrollNext = useCallback(() => {
+    if (embla) embla.scrollNext();
+  }, [embla]);
+
+  const emblaSx = {
+    overflow: "hidden",
+    ...emblaSxOuter,
+  };
+  const emblaContainerSx = {
+    display: "flex",
+    width: "100%",
+  };
+
+  const dotIconButtonSx = {
```

Comment:
Ok, I can see now, that this component keepts too much styles, which does not belong to it. Let's than move dot styles to DotButton, but do not inline them, leave as variables.